### PR TITLE
Bug 1157945 - Support newly added g2 tests

### PR DESCRIPTION
**Pull request by KWierso**
[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/481)

### CODE REVIEW

**Comment by KWierso** [comment]
Fixed the Travis failure. Used a ) instead of a } on accident.

File: tests/etl/test_buildbot.py [hunk 1]
```diff
@@ -1274,6 +1285,17 @@
...
+     {'build_type': 'opt',
+      'job_type': 'talos',
+      'name': {'group_name': 'Talos Performance e10s',
+               'group_symbol': 'T-e10s',
+               'name': 'Talos g2 e10s',
+               'job_symbol': 'g2'},
+      'platform': {'arch': 'x86_64',
+                   'os': 'mac',
+                   'os_platform': 'osx-10-6',
+                   'vm': False}}),
```
Comment (jmaher) [inline tests/etl/test_buildbot.py]: note: we turned off all 10.6 talos tests- we won't be scheduling this on 10.6, so this clause might be optional.

**Comment by edmorley** [comment]
The test is failing because the regex doesn't handle the os x 10.6 '-snow' case, however from Joel's comment it sounds like we don't need to support it. Could you change the test to be for another OS X version (or even another OS) - so long as it tests the e10s variant of the buildername :-)

**Comment by KWierso** [comment]
Looks fixed :)

**Pull request merged**